include/Jit/EEObjectLinkingLayer.h

```diff
@@ -34,15 +34,15 @@ class EEObjectLinkingLayer : public ObjectLinkingLayerBase {
   class ConcreteLinkedObjectSet : public LinkedObjectSet {
   public:
     ConcreteLinkedObjectSet(MemoryManagerPtrT MemMgr,
-                            SymbolResolverPtrT Resolver)
-        : LinkedObjectSet(*MemMgr, *Resolver), MemMgr(std::move(MemMgr)),
-          Resolver(std::move(Resolver)) {}
+                            SymbolResolverPtrT Resolver,
+                            bool ProcessAllSections)
+        : LinkedObjectSet(*MemMgr, *Resolver, ProcessAllSections),
```

Comment:
This line is the critical one for the `ProcessAllSections` stuff -- the base `LinkedObjectSet` class is not a llilc-specific copy and its constructor now takes a `ProcessAllSections` parameter.  I believe just passing `false` here would have been correct, but copied the full change for the reasons I mentioned in my previous comment.